cram_common/cram_plan_occasions_events/src/default-plan-events.lisp

```diff
@@ -108,6 +108,29 @@
                 'simple-error
                 :format-control "OBJECT-ARTICULATION-EVENT requires an opening distance."))))
 
+(defclass environment-manipulation-event (event)
```

Comment:
Docstrings added. Didn't comment out the other event yet, because there are some references in old code. I guess they don't matter anymore, but I'm not sure. Maybe you can look into it.